operator/pkg/manifest/shared.go

```diff
@@ -383,6 +383,8 @@ func RewriteURLToLocalInstallPath(installPackagePath, profileOrPath string, skip
 		return "", "", err
 	}
 	if isURL {
+		// TODO: remove this feature in next major release
+		log.Warnf("fetching charts from arbitrary URLs will be removed in next major released")
```

Comment:
Nit:

```Fetching charts from arbitrary URLs will be removed in the next major release```

Reply:
@zirain I am not exactly sure what arbitrary URLs means, from looking at the code, it seems to me that any URL will not be supported. Do we have `arbitrary` URL vs `non arbitrary` or `official` URLs? If not, I would think this is basically saying we no longer support using URLs?

Reply:
The intent is no URLs. Users should use local files. Fetching arbitrary tarballs from the internet and then applying them as root to the cluster is problematic on many levels, and not actually a use case istio users really have; they apply local files or built in manifests

Reply:
@zirain @howardjohn right, in that case, we should simply say URL pointed charts wont be supported or deprecated in the future. `arbitrary` just seems to me not very clear.

Reply:
remove `arbitrary` from log